infrastructure/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   passivbot:
-    build: .
+    build: ..
```

Comment:
The volume will be breaked because we changed the directory of docker-compose.yml file.
Maybe it's not worth to change the location of Dockerfile and docker-compose.yml, it add complexity in the resolution of the paths for only two files.